### DISCUSSION TITLE
New UI Design for Laragon Website?

### DISCUSSION THREAD

**Discussion by gurjarshakti** [leokhoa/laragon#972] [Ideas]
Hi Team, 
I am exploring Laragon from the last two and three days. I love this. I want to contribute to this project as a UI/UX Designer.

Let me know your thoughts on it.

**Comment by jabbarn** [comment]
Hi, I would like to help improve the layout as well.

**Comment by gurjarshakti** [comment]
Nice to meet you @DaemonicNaymon but the project owner is no more active on this project. I assume. Maybe

**Comment by jabbarn** [comment]
Hi @gurjarshakti, nice to meet you aswell. I'm sure the project owner more active at other issues.

**Comment by Wulfheart** [comment]
@gurjarshakti If your design is not that complicated I would help you by covering the dev side.

**Comment by Bam92** [comment]
Hi designers! 

I'm not a designer like you, but a dev. But, I've also noticed that the project lacks a good UI. More than 5 mo ago the project owner does not respond. Are they still active? Or maybe the project needs to be forked?

**Comment by AWqxKAWERbXo** [comment]
Hey folks, still using Laragon? If anybody still wants to contribute, please answer here even the issue is closed. I'm sure @leokhoa will take note and knows best whether and where he needs support.

**Comment by Sophist-UK** [comment]
@gurjarshakti Shakti

I am sure that we would all love to see some wireframes of any ideas you might have for a general UI improvement. But there are probably more things we might like to see - and here is my braindump:

* Centralised log files, plus a widget showing a consolidated tail of a selected subset of those log files.
* Ability for menu items (and their sub-items) to be defined using ini files and for such a menu item to run a CMD or PowerShell or Bash script and for the script output i.e. progress to be piped into another similar widget.
* Generally more modern design
* Ability to easily define which services should be started and stopped as a group, and (like other scripts) for the status of each service starting up to be shown in a consolidated widget until all services are started.

Note for @leokhoa : This is where you should be focussing your own time - plus managing the to-do-list and providing support, whilst you allow senior community members to contribute and help you with the menu ini files, scripts, adding new services, improving existing services, doing documentation etc. all through the coordinating functionality provided by the Laragon executable.

**Comment by Sophist-UK** [comment]
@alxndr-w Alexander - Please reopen this and transfer it to Discussions so that we can keep synergising here.

**Comment by Sophist-UK** [comment]
See also #796.

**Comment by AWqxKAWERbXo** [comment]
👀 It's possible to move issues to discussions? Didn't know that until now. Sure!

**Comment by AWqxKAWERbXo** [comment]
You might want to know that @leokhoa published a new website: <https://laragon.org/>

**Comment by Sophist-UK** [comment]
I am a bit confused - a lot of this discussion is about UI/UX for Laragon itself, but the title and some comments are about web site design for the Laragon marketing web site.
